### PR TITLE
feat: page layout desktop version

### PR DESCRIPTION
**Pull request by ekremney**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/rolex-event/issues/3

Test URLs:
- Before: [https://main--rolex-event--hlxsites.hlx.page/en/evening-events](https://main--rolex-event--hlxsites.hlx.page/en/evening-events)
- After: [https://page-layout--rolex-event--hlxsites.hlx.page/en/evening-events](https://page-layout--rolex-event--hlxsites.hlx.page/en/evening-events)

Please also see the content structure change in the sharepoint /en/evening-events.docx file

- Grid layout used as per [rolex intranet site](http://rlxntr2022.ocitocine.com/pages/layouts/intranet/who.html) where main content is on the left and secondary content is on the right
- hero block converted to regular block and related js functions removed
- various styling (ie border radiuses) added as well

Screen Shot 2022-11-24 at 16 05 13

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -330,9 +330,42 @@ a.button.secondary:hover, a.button.secondary:focus {
   cursor: pointer;
 }
 
+main {
+  display: grid;
+  grid-template-columns: 1fr 360px;
+  grid-template-areas:
```
Comment (ramboz) [inline styles/styles.css:336]: maybe changing this to:
```
hero aside
main-content aside
```
and then setting the h1's grid-area to `hero` would let you keep the existing structure

File: scripts/scripts.js [hunk 2]
```diff
@@ -66,7 +41,6 @@ export function decorateMain(main) {
   // hopefully forward compatible button decoration
   decorateButtons(main);
   decorateIcons(main);
-  buildAutoBlocks(main);
```
Comment (solaris007) [inline scripts/scripts.js:69]: so for Rolex we're completely abandoning auto-blocking which is stipulated by Helix as the ideal case as it facilitates unconstrained authoring? @ramboz

Comment (solaris007) [reply scripts/scripts.js:69]: for reference: https://www.hlx.live/developer/markup-sections-blocks#auto-blocking

File: scripts/scripts.js [hunk 3]
```diff
@@ -44,19 +32,6 @@ export function wrapImgsInLinks(container) {
   });
 }
 
-/**
- * Builds all synthetic blocks in a container element.
- * @param {Element} main The container element
- */
-function buildAutoBlocks(main) {
```
Comment (ramboz) [inline scripts/scripts.js:51]: I would leave the auto blocks, just maybe disable the hero one if it has undesired side-effects on our grid

File: styles/styles.css (hunk 1 above)
Comment (ramboz) [reply styles/styles.css:336]: It should just be a matter of fixing:
- the hero section width
- the right top rounded border on it
- offsetting the aside top position so it aligns with the rest

File: styles/styles.css [hunk 4]
```diff
@@ -330,9 +330,42 @@ a.button.secondary:hover, a.button.secondary:focus {
...
+.main-content {
+  grid-area: main-content;
+  padding: 72px 20px 60px 40px;
+  width: 100%;
+  border-top-left-radius: 8px;
+}
+
+.aside {
+  grid-area: aside;
+  margin: 0 auto !important;
```
Comment (ramboz) [inline styles/styles.css:357]: you shouldn't have to use `!important`… ever 😉 Unless you really know what you are doing. You can probably just increase the selector specificity to achieve the desired effect

File: styles/styles.css [hunk 5]
```diff
@@ -420,7 +449,6 @@ hr {
   }
 
   .section > div {
-    max-width: 1060px;
```
Comment (ramboz) [inline styles/styles.css:423]: Any reason for this? Their intranet design has a forced max width for regular content

File: styles/styles.css [hunk 6]
```diff
@@ -397,10 +430,6 @@ hr {
 
 /* small screen laptops */
 @media all and (min-width: 600px) {
-  main .section {
-    padding: 0 32px;
```
Comment (ramboz) [inline styles/styles.css:401]: Any reason for this?

**Comment by ekremney** [comment]
I changed the grid layout to the one we discussed in the call. Hero is being autoblocked now. @ramboz will add autoblocking for the aside content as well
cc @solaris007 @ramboz

File: styles/styles.css (hunk 1 above)
Comment (ekremney) [reply styles/styles.css:339]: done!

**Review by ramboz — APPROVED** [review]

**Review by solaris007 — APPROVED** [review]

**Pull request merged**